### DISCUSSION TITLE
Translations: add Persian language to translations

### DISCUSSION THREAD

**Discussion by MahdiTa97** [saadeghi/daisyui#1259] [Ideas / feature requests]
Please add the **Persian** language. So we can see how **RTL** functionality works in Daisyui.

**Comment by saadeghi** [comment]
Moving this to discussions since it's not an issue.  
PRs are welcome. You can add any translation [here](https://github.com/saadeghi/daisyui/tree/master/src/docs/src/translation)

But if you just want to see how RTL looks like, here's an example config: https://play.tailwindcss.com/fIwm2ZlzPJ?file=config
